src/components/type-utils.ts

```diff
@@ -107,3 +107,10 @@ export function extractDeclaration(symbol: ts.Symbol) {
   }
   return declarations[0];
 }
+
+export function printFlags(flags: number, mapping: Record<number, string>) {
```

Comment:
Not used in the final version of the code, but it was useful in local debugging to convert binary flags into readable strings

Let's keep it in the code so I will not have to remember how it is done every time